core/geometry/src/clipping/ClipUtils.ts

```diff
@@ -490,6 +492,38 @@ export class ClipUtilities {
     }
     return false;
   }
+
+  private static _workTransform?: Transform;
```

Comment:
optional: move these 3 private variables to the top of the class (although I dont know what the standard is for where to put private variables; top of the class or close to where they are used)

Reply:
Good idea. There's no standard I'm aware of. But having general purpose statics up there below any instance data increases their visibility and thus encourages their (judicious) re-use in methods. Of course it would have been nicer if we could have static vars _inside_ a method, but that's not happening.

Reply:
> it would have been nicer if we could have static vars inside a method, but that's not happening.

You [can](https://www.typescriptlang.org/play?#code/MYGwhgzhAEAqAWBLAdgc2gbwFDV9ADgK4BGIiw0qApgC4ByVAHjQBQCUAXNMoQLbFUATphx4xAegBU0CDTA1y0SeOgha0YIUGCqyGtAC80AAwBuUWNwIUqAHT5BAexrOAnviq3q9JvqPtDAD4NLR09AGpw80s8HRotZGgaJAgvWgZmdmi8AF8sPKxgR2RZaDBDbioAdzgkNCzC4tLiCuRq2psGrHEVAAVBFBoYACJjABpoAEYJgCYJgGZhxpLHNVsQR1QWMDSfTLYJ4l2M1gPoI+8T9gmdy992NlMgA) but you shouldn't.

Reply:
I needed this in my day.